code/modules/wiremod/components/list/index.dm

```diff
@@ -26,11 +26,11 @@
 	var/index = index_port.value
 	var/list/list_input = list_port.value
 
-	if(!islist(list_input) || !index)
+	if(!islist(list_input) || !isnum(index))
 		output.set_output(null)
 		return
 
-	if(isnum(index) && (index < 1 || index > length(list_input)))
+	if(index < 1 || index > length(list_input))
```

Comment:
actually I think this is meant to accept string indexes? The port type of index is "any"... ?

Reply:
yeah idk about that